### DISCUSSION TITLE
Opensearchstack helm chart missing

### DISCUSSION THREAD

**Discussion by stoeps13** [HCL-TECH-SOFTWARE/connections-automation#280] [General]
Hi,
as opensearch is configured to write security-auditlogs, I wanted to try Opensearch dashboards.

This repo has a variable setup_opensearchstack which defaults to false, but even so, the helmvars file is written, but the installation fails, because the helm chart opensearchstack is missing.

Are there plans to provide a helm chart for opensearchstack? Any ETA?

Is there a config option to disable the security-auditlogs or prune indices older n days?

Thanks in advance
Stoeps

**Comment by stoeps13** [comment]
Opensearch creates a security-auditlog for each day, it seems there is nothing which deletes them.

![image](https://github.com/HCL-TECH-SOFTWARE/connections-automation/assets/1975258/79f16d59-cc33-4b11-bdaa-52ee4cf1bcd1)

@sabrina-yee do you know if there are plans to support dashboards / opensearchstack in a future version? Any information why security-auditlogs are enabled?

Thanks
Stoeps

**Reply by sabrina-yee** [reply]
@stoeps13 yes, we're evaluating the work needed to support the opensearchstack.
